lib/rtc_peer.js

```diff
@@ -116,8 +116,8 @@ export class RTCPeer extends EventEmitter {
         this.logger.logDebug(`RTCPeer: ICE connection state change -> ${(_a = this.pc) === null || _a === void 0 ? void 0 : _a.iceConnectionState}`);
     }
     onNegotiationNeeded() {
+        var _a, _b;
```

Comment:
That's interesting. Change to the tsc?

Reply:
:man_shrugging:

Reply:
Looks like it's failing the build check, so might want to run it again and see if it fixes?